app/views/layouts/base.html.erb

```diff
@@ -15,6 +15,7 @@
     <%= yield(:stylesheets) %>
 
     <%= csrf_meta_tags %>
+    <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js'), "data-turbolinks-track" => true %>
```

Comment:
it wont, you need to include it in your plugin.
e.g, in a page somewhere (either in the layout or a specific page) add:
```erb
+<%= javascript_include_tag *webpack_asset_paths('name of chunk'', :extension => 'js'), "data-turbolinks-track" => true %>
```